FoodPin/FoodPin/View/RestaurantDetailTwoColumnCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  RestaurantDetailTwoColumnCell.swift
+//  FoodPin
+//
+//  Created by Viviana Mesaros on 27.04.2022.
+//
+
+import UIKit
+
+class RestaurantDetailTwoColumnCell: UITableViewCell {
+
+    @IBOutlet private var column1TitleLabel: UILabel! {
+        didSet {
+            column1TitleLabel.text = column1TitleLabel.text?.uppercased()
+            column1TitleLabel.numberOfLines = 0
+        }
+    }
+
+    @IBOutlet private var column1TextLabel: UILabel! {
+        didSet {
+            column1TextLabel.numberOfLines = 0
+        }
+    }
+
+    @IBOutlet private var column2TitleLabel: UILabel! {
+        didSet {
+            column2TitleLabel.text = column2TitleLabel.text?.uppercased()
+            column2TitleLabel.numberOfLines = 0
+        }
+    }
+
+    @IBOutlet private var column2TextLabel: UILabel! {
+        didSet {
+            column2TextLabel.numberOfLines = 0
+        }
+    }
+
+    func setColumn1TitleLabel(text: String) { self.column1TitleLabel.text = text }
+    func setColumn1TextLabel(text: String) { self.column1TextLabel.text = text }
+    func setColumn2TitleLabel(text: String) { self.column2TitleLabel.text = text }
+    func setColumn2TextLabel(text: String) { self.column2TextLabel.text = text }
```

Comment:
Number of lines can be set from storyboard and you can make text uppercased in the setColumn methods, where you initially set the text.
Also, the name of the labels are not very intuitive, what are those label actually holding, a title, description, name?